src/LLMProviders/embeddingManager.ts

```diff
@@ -46,6 +46,7 @@ export default class EmbeddingManager {
       embeddingModel,
       openAIEmbeddingProxyBaseUrl,
       openAIEmbeddingProxyModelName,
+      useOpenAILocalProxy,
```

Comment:
Aren't we removing the local proxy in this PR? I thought this flag was deprecated because of that?